### ISSUE TITLE
HTML Tags appear in update dialogue

### ISSUE THREAD

**Issue by mokkin** [nextcloud/desktop#2200]
### Expected behaviour
Clear links

### Actual behaviour
HTML Tags appear in update dialogue

### Steps to reproduce
1. boot system

### Client configuration
Client version:

Operating system:
Ubuntu 20.04
OS language:
german
Qt version used by client package (Linux only, see also Settings dialog):

Client package (From Nextcloud or distro) (Linux only):
Nextcloud PPA
v2.6.4

### Server configuration

Nextcloud version:

Storage backend (external storage):

### Logs

![DeepinBildschirmfoto_Bereich auswählen_20200714100534](https://user-images.githubusercontent.com/2938748/87442961-2fede280-c5f5-11ea-948b-aab71dedb2b6.png)

**Comment by nicolasfella** [comment]
Is this a desktop notification?

In principle it's legal to have HTML in there ([see spec](https://specifications.freedesktop.org/notification-spec/notification-spec-latest.html#markup), however not all servers support it. For servers that do not have the `body-markup` capability we would need to strip away the HTML.

**Comment by mokkin** [comment]
> Is this a desktop notification?

Yes, indeed.

**Comment by nicolasfella** [comment]
@mokkin Are you using the GNOME desktop or something else?

**Comment by nicolasfella** [comment]
Relevant line is here: https://github.com/nextcloud/desktop/blob/master/src/gui/updater/ocupdater.cpp#L151

It strikes me as odd that the URL looks clickable as if it is correctly interpreted as a URL by the notification server. Could it be that `_updateInfo.web()` already contains the a tag and we end up having it twice?

**Comment by mokkin** [comment]
> @mokkin Are you using the GNOME desktop or something else?

Yes, GNOME Shell 3.36.3 on Ubuntu 20.04

**Comment by danielcotton** [comment]
This is also an issue on macOS (see attached screenshot). It also doesn't appear to have a click action, so there's no way to actually update via the notification.
<img width="339" alt="Screen Shot 2020-08-04 at 6 25 05 pm" src="https://user-images.githubusercontent.com/31445918/89274521-2e598c80-d680-11ea-8467-3d01cf5d1632.png">

**Comment by sberegszaszi** [comment]
> Could it be that _updateInfo.web() already contains the a tag and we end up having it twice?

@nicolasfella I don't think so. I checked the log file when running 2.6.4 version and the download link in the xml file that is requested when checking for updates does not contain the html tag.
On Ubuntu I get the same result, link not showing correctly, but when running the same 2.6.4 version on Manjaro XFCE, the desktop notification showes correctly.
![Screenshot_2020-08-07_15-50-24](https://user-images.githubusercontent.com/51827363/89654638-3bb58780-d8c9-11ea-8d04-d5c0ee6ee800.png)

**Comment by danielcotton** [comment]
Can confirm this is still happening on macOS in 3.1.0

**Comment by danielcotton** [comment]
People are clearly still having the issue per #3005, so the issue should remain open

**Comment by Valdnet** [comment]
@FlexW There is still a problem with this text string:

https://github.com/nextcloud/desktop/blob/104d3c0661afe459bed34c1ff30c9f3c98b75120/src/gui/updater/ocupdater.cpp#L156
 
that appears in new releases. See: #3005 and [help.nextcloud.com](https://help.nextcloud.com/t/problem-with-linking-polish-macos-desktop-client/107526/7).

No fix has been released for **eight months**. If we continue to fix bugs for so long, we can expect users to stop reporting bugs and the application to stop developing.

**Comment by rakekniven** [comment]
> People are clearly still having the issue per #3005, so the issue should remain open

The original issue is open.

**Comment by FlexW** [comment]
@Valdnet thanks for pointing this out. I know there are some pretty old bug reports but I'm just for two and a half months on the team. I try to look into this next week.

**Comment by Valdnet** [comment]
@FlexW Thanks for your interest in the problem. We are waiting for a fix 😉.

**Comment by rakekniven** [comment]
Please vote at the initial postin.

**Comment by danielcotton** [comment]
> The original issue is open.

@rakekniven This is the original issue. I commented so it wouldn't be auto-closed.

**Comment by rakekniven** [comment]
> I commented so it wouldn't be auto-closed.

@danielcotton Understood. Please vote at the initial posting at this issue to make it more important.

**Comment by FlexW** [comment]
@Valdnet I created a pull request to resolve the issue. Please read my comment there. https://github.com/nextcloud/desktop/pull/3052

**Comment by rakekniven** [comment]
Hello @FlexW , 

your PR does not solve this issue directly.
You remove the hyperlink. Now the user has to copy the URL and open it inside a browser, or?

**Comment by FlexW** [comment]
See also this follow up issue https://github.com/nextcloud/desktop/issues/3054

**Comment by rakekniven** [comment]
> Please see the comment in the pr on how that issue should be solved correct in my opinion and why I don't do it now.

Sorry, overseen it. Thumbs up!
